bin/checkVersion.js

```diff
@@ -1,6 +1,7 @@
 /*To avoid dependencies, the compareVersions code below has been copied from: https://github.com/omichelsen/compare-versions*/
 /*eslint no-console: "off"*/
 /*eslint no-useless-escape: "off"*/
+/*eslint indent: "off"*/
```

Comment:
Is this rule even enabled?

Reply:
I may have had that enabled locally...Let me remove it.  Do you want me to delete this branch and submit a new PR so everything is a little cleaner?